### PR TITLE
v1.6.1 release notes fixes

### PR DESCRIPTION
**Pull request by xaur**

### CODE REVIEW

**Review by alexlyp — APPROVED** [review]

**Comment by xaur** [comment]
I'll produce a reflown version for GitHub release page too.

**Comment by xaur** [comment]
Here is a version for GitHub release page with joined lines, added anchor links and the skip to downloads link:

https://raw.githubusercontent.com/xaur/decred-binaries/ghrelease/github/release-v1.6.1.md

It is reado for copy-pasting straight into the release page editor. I made a [test release](https://github.com/xaur/decred-binaries/releases/tag/v1.6.1) in my fork to test that it renders correctly.

To verify that I didn't insert anything you can follow how it was produced here:

https://github.com/xaur/decred-binaries/commits/ghrelease/github/release-v1.6.1.md

Basically, I copied the text from the last commit in this PR (288eea9), made a few tweaks and joined lines with regex replacement `([^\n|:]) *\n *([^\n\*-])` to `\1 \2`.

**Review by alexlyp — APPROVED** [review]
